firebase-database/src/androidTest/java/com/google/firebase/database/QueryTest.java

```diff
@@ -2079,6 +2455,43 @@ public void startAtWithTwoArgumentsWorksCase1169()
     assertEquals(2, i);
   }
 
+  @Test
+  public void startAfterWithTwoArgumentsWorksCase1169()
```

Comment:
The case number looks like it refers to an old bug about startAt, which is not applicable here. I would drop it (at least from here)